{
  "id": "CVE-2018-1000805",
  "elsa_id": "ELSA-2018-3347",
  "definition_id": "oval:com.oracle.elsa:def:20183347",
  "title": "\nELSA-2018-3347:  python-paramiko security update (CRITICAL)\n",
  "description": "\n[2.1.1-9]\n- Fix a security flaw (CVE-2018-1000805) in Paramiko's server\n  mode (does not effect client mode).\n  Backported from 2.1.6\n  Resolves rhbz#1637366\n",
  "severity": "CRITICAL",
  "affected": {
    "family": "unix",
    "platform": "Oracle Linux 7"
  },
  "packages": [
    {
      "name": "python-paramiko",
      "fixed_version": "0:2.1.1-9.el7",
      "arch": "aarch64"
    },
    {
      "name": "python-paramiko-doc",
      "fixed_version": "0:2.1.1-9.el7",
      "arch": "aarch64"
    },
    {
      "name": "python-paramiko",
      "fixed_version": "0:2.1.1-9.el7",
      "arch": "x86_64"
    },
    {
      "name": "python-paramiko-doc",
      "fixed_version": "0:2.1.1-9.el7",
      "arch": "x86_64"
    }
  ],
  "references": [
    {
      "id": "ELSA-2018-3347",
      "source": "elsa",
      "url": "https://linux.oracle.com/errata/ELSA-2018-3347.html"
    },
    {
      "id": "CVE-2018-1000805",
      "source": "CVE",
      "url": "https://linux.oracle.com/cve/CVE-2018-1000805.html"
    }
  ],
  "issued": "2018-11-07T00:00:00Z",
  "rights": "Copyright 2018 Oracle, Inc."
}
